nucliadb_protos/dataset.proto

```diff
@@ -38,6 +38,7 @@ message Label {
 message TextLabel {
     string text = 1;
     repeated Label labels = 2;
+    string id = 3;
```

Comment:
should this be named `field_id` ?

Reply:
No, it serves as field_id and paragraph_id. It's used in both classification protobuffers